tests/Fixtures/inc/classes/subscriber/Tools/CacheDirSizeCheckSubscriber.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace WP_Rocket\Tests\Fixtures\inc\classes\subscriber\Tools;
+
+use WP_Rocket\Subscriber\Tools\Cache_Dir_Size_Check_Subscriber;
+
+/**
+ * Allow to test the class Cache_Dir_Size_Check_Subscriber.
+ */
+class CacheDirSizeCheckSubscriber extends Cache_Dir_Size_Check_Subscriber {
```

Comment:
what is this used for?

Reply:
It was to prevent the test to use `get_dir_size()`, but it doesn’t seem to be an issue indeed.